proto/api.proto

```diff
@@ -24,10 +24,6 @@ import "cockroach/proto/data.proto";
 import "cockroach/proto/errors.proto";
 import "gogoproto/gogo.proto";
 
-option (gogoproto.sizer_all) = true;
-option (gogoproto.marshaler_all) = true;
-option (gogoproto.unmarshaler_all) = true;
```

Comment:
I'm not sure if this is an improvement. Sure, you've removed many of these options, but not all of them. And now a developer has to remember that we're using a custom protoc plugin. Forgetting to add these options to a file would suck, but perhaps a centralized check would capture that while leaving the code explicit.